client/src/elm/Utils/NominalDate.elm

```diff
@@ -129,7 +129,7 @@ renderDate language date =
     )
         ++ " "
         ++ month
-        ++ ", "
+        ++ " "
```

Comment:
No need for all these ++. Lets make it `++ " month " ++`.

Reply:
@Diama1 ^^

Reply:
> 
@anvmn but month is not a string, it is a variable

Reply:
@Diama1 right, my mistake.
Sorry.